packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DatePickerInput.tsx

```diff
@@ -13,10 +13,34 @@ export type DatePickerInputProps = {
   onChange: (date?: Date) => void;
   parseInput: (text: string) => Date;
   formatDate: (date: Date) => string;
+  /**
+   * Decides if this component is used for the 'from' or 'to' date
+   */
+  isFromOrTo?: 'from' | 'to';
```

Comment:
Is this done for a reason or just because?
I think `isStartDate = true` for "from" and false for "to" could be enough.

Reply:
see history: https://github.com/iTwin/iTwinUI/pull/1211#discussion_r1172602567

there are many ways to do this. one option is to have two props `upperBound`/`lowerBound` that simply accept dates.
```
<DatePickerInput lowerBound={new Date()} />
```
```
<DatePickerInput upperBound={new Date()} />
```
or even combine the two props into one (questionable lol)
```
<DatePickerInput bound={{ type: 'lower', date: new Date() }} />
```

Reply:
Maybe `useAs? : 'from' | 'to'` sound better than isFromOrTo?

Reply:
- too vague
- `use` prefix is for hooks
- `as` is generally used to control the element which it is rendered as (polymorphism)

ultimately it doesn't matter because this component is private, so i'm fine with all of the suggestions. My top preference is for `upperBound`+`lowerBound` (remove `isFromOrTo` and `selectedDate`).

@gretanausedaite Maybe you can also list your top preference, and then we can let @adamhock make the final decision?

Reply:
My preferences:
- leave as is
- Mayank's suggestion also sounds good.